src/codewhisperer/util/supplementalContext/crossFileContextUtil.ts

```diff
@@ -175,6 +195,35 @@ function getOpenFilesInWindow(): string[] {
     return filesOpenedInEditor
 }
 
+/**
+ *
+ * @returns file distance between targetFile and candidateFile
+ * For example:
+ * The file distance between A/B/C.java and A/B/D.java is 0
+ * The file distance between A/B/C.java and A/D.java is 1
+ */
+export function getFileDistance(targetFile: string, candidateFile: string, seperator: string): number {
```

Comment:
Don't need a separator arg. Should normalize() the path anyway, because things like double slashes `//` should be collapsed to a single slash `/`, and other quirks. See `isInDirectory` : https://github.com/aws/aws-toolkit-vscode/blob/d66f115ee4e1fed8d87fa6a59bd83a1c5925ac05/src/shared/filesystemUtilities.ts#L135

Reply:
yea sgtm